### ISSUE TITLE
Can't install nvidia-docker rpm after installing CentOS 7 default docker package

### ISSUE THREAD

**Issue by mnogu** [NVIDIA/nvidia-docker#102]
nvidia-docker-1.0.0.rc.2-1.x86_64.rpm requires docker-engine:
```
Requires: docker-engine
```
https://github.com/NVIDIA/nvidia-docker/blob/v1.0.0-rc.2/tools/build/rpm/SPECS/nvidia-docker.spec#L20

And [docker-engine in the yum.dockerproject.org repository](https://docs.docker.com/engine/installation/linux/centos/) conflicts with the default docker package in the CentOS 7 Extras repository:
```
Error: docker-engine-selinux conflicts with docker-selinux-1.9.1-40.el7.centos.x86_64
Error: docker-engine conflicts with docker-1.9.1-40.el7.centos.x86_64
```

So, when the default docker package is already installed in a GPU server, we can't install the nvidia-docker rpm into the server.

Is the use of the default docker package discouraged?

**Comment by 3XX0** [comment]
We recommend the official Docker installation procedure, hence the `Requires: docker-engine`.
Moreover, CentOS official repository not only has `docker` but also `docker-latest` and both of them are outdated.

**Comment by mnogu** [comment]
I confirmed that a GPU application works with `docker` and `nvidia-docker_1.0.0.rc.2_amd64.tar.xz` by following [the install instruction for other distributions](https://github.com/NVIDIA/nvidia-docker/tree/v1.0.0-rc.2#other-distributions). And after uninstalling `docker` and installing `docker-engine-1.11.1-1.el7.centos.x86_64.rpm`, I also confirmed that a GPU application works with `nvidia-docker-1.0.0.rc.2-1.x86_64.rpm`.

Thank you for this great tools.
